error-handlers.js

```diff
@@ -0,0 +1,27 @@
+exports.psqlErrorHandler = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Bad request" });
+  } else {
+    next(err);
+  }
+};
+
+exports.customErrorHandler = (err, req, res, next) => {
+  if (err.msg && err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+};
+
+exports.serverErrorHandler = (err, req, res, next) => {
+  res.status(500).send({ msg: "500: Internal server error!" });
```

Comment:
I'd recommend a back up console log in here for any error that gets here. Would be useful for debugging to know why.